ProjectLighthouse/Database.cs

```diff
@@ -312,6 +317,40 @@ public async Task UnqueueLevel(int userId, Slot queuedSlot)
         await this.SaveChangesAsync();
     }
 
+    public async Task BlockUser(int userId, User blockedUser)
+    {
+        if (userId == blockedUser.UserId)
+        {
+            return;
+        }
```

Comment:
```suggestion
        if (userId == blockedUser.UserId) return;
```
I would inline this to match the styling in `UnblockUser`